Await page rebased

Filipo0on opened the pull request:
madenew branch rebased from develop

malinhanak commented on client/src/components/lobby/LobbyAwaitGame.tsx:
```diff
@@ -0,0 +1,215 @@
+import styled from 'styled-components';
+import BGImage from '../../dist/images/chess-bg.jpg';
+import lobbyStore from "src/store/LobbyStore";
+const mockGame = {creator : "Anonymous", creatorColor: "",  opponent : "Anonymous", gameId : 2, gameType : "Classic", gameTime : 5, gameAddTime : 15, gameStarted : false, player1Ready: false, player2Ready: false,}; 
```
Why are you not importing the MockData from the store MockData file?

Filipo0on commented:
I have now updated the await page to the more simpel version that we agreed on before. 

WHAT HAVE BEEN DONE

- Removed logic for checking if player is 1 or 2
- Removed logic for making player ready/reading if other player is ready
- AwaitPage now checks if "gameStarted" is true. if it is the page displays "joining game" for 1.5 sec, and ther routs the player to the game ( /game/id)
- Adjusted the code according to the comments on the PR (the ones that still apply after the removing of some functions) 

SCREENSHOTS

Waiting for Opponent:

![prbild2](https://user-images.githubusercontent.com/31758737/51027955-755dc980-1592-11e9-9b62-6247e0d9ea06.png)


Joining Game: 

![prbild1](https://user-images.githubusercontent.com/31758737/51027973-80185e80-1592-11e9-8d8b-bd9fd06035c0.png)
